pkg/notifier/factory.go

```diff
@@ -22,14 +22,16 @@ import (
 
 type Factory struct {
 	URL      string
+	APIToken string
 	ProxyURL string
 	Username string
 	Channel  string
 }
 
-func NewFactory(url string, proxy string, username string, channel string) *Factory {
+func NewFactory(url string, apiToken string, proxy string, username string, channel string) *Factory {
```

Comment:
@stefanprodan I removed the `NewSlackApiFactory` and added `apiToken` in here instad. Would that cause any issues?

Reply:
I see no issues with it.